src/styles/ProgressBarStyle.tsx

```diff
@@ -0,0 +1,48 @@
+import { StyleSheet } from "react-native";
+
+export const styles = StyleSheet.create({
+  container: {
+    // flex: 1,
+    height: 80,
+    backgroundColor: '#eee',
+    borderRadius: 10,
+    margin: 10,
+    padding: 10,
+  },
+  bar: {
+    height: '30%',
+    backgroundColor: '#3498db',
+    borderRadius: 10,
+    alignItems: 'center',
+  },
+  progressContainer: {
+    alignItems: 'center',
+    marginTop: 10,
+  },
+  progressText: {
+    fontSize: 16,
+    fontWeight: 'bold',
+    color: 'black',
+  },
+  buttonsContainer: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    marginTop: 5,
+    textAlign: 'center',
+  },
+  button: {
+    backgroundColor: '#2980b9',
+    borderRadius: 5,
+    // padding: 10,
+    height: 30,
+    width: 30,
```

Comment:
use scale function, for height width for responsivness

Reply:
Will Fix this in another ticket